autoPyTorch/api/base_task.py

```diff
@@ -920,6 +941,8 @@ def _search(
                 self._stopwatch.stop_task(traditional_task_name)
 
         # ============> Starting ensemble
+        self.precision = precision
+        self.opt_metric = optimize_metric
```

Comment:
they are being used when fitting an ensemble post hoc. We require this to allow the user to reuse the values passed for search in fit_ensemble in case both are None.

Reply:
This comment also should be addressed.